Add download links to the download page (#118)
void-linux/void-linux.github.io: _platforms/sbc.md

---
name: arm platforms
date: 1970-01-01 05:00:00
devices:
- name: beaglebone
  arch: armv7
- name: cubieboard2
  arch: armv7
- name: odroid-c2
  arch: armv7
- name: rpi
  arch: armv6
- name: rpi2
  arch: armv7
- name: rpi3
  arch: aarch64
- name: usbarmory
  arch: armv7
---

{% capture download_details %}

> Live images can be written onto an SD card (i.e. using `dd`) and they allow you to have a ready to boot system. These images are prepared for 2GB SD cards. Alternatively, use the rootfs tarballs if you want to customize the partitions and filesystems.

Reviewer: ```suggestion
Live images can be written onto an SD card (i.e. using `dd`) and they provide you with a ready to boot system. These images are prepared for 2GB SD cards. Alternatively, use the ROOTFS tarballs if you want to customize the partitions and filesystems.
```

Could `ROOTFS tarballs` be a link to the `arm` page, perhaps?

> Connect to it using a virtual terminal or via ssh and log in as `root/voidlinux`.

Reviewer: ```suggestion
Connect to the system using a virtual terminal or SSH and log in as `root` with password `voidlinux`.
```
